### PR TITLE
enable vm's to be managed by ansible

### PR DESCRIPTION
**Pull request by pradeeponnada**

### CODE REVIEW

File: modules/WindowsScriptExtension_v1/main.tf [hunk 1]
```diff
@@ -10,25 +10,28 @@ NOTES
...
+resource "azurerm_virtual_machine_extension" "ScriptExtension" {
+  name                       = "ScriptExtension"
+  virtual_machine_id         = var.virtual_machine_id
+  publisher                  = "Microsoft.Compute"
+  type                       = "CustomScriptExtension"
+  type_handler_version       = "1.10"
+  auto_upgrade_minor_version = true
+  protected_settings         = <<PROTECTED_SETTINGS
+    {         
+            "fileUris": ["https://ansibleteststrgaccnt.blob.core.windows.net/winrmext/InitializeDisks.ps1",
```
Comment (CHolmesAtLVS) [inline modules/WindowsScriptExtension_v1/main.tf:29]: Can you make the path to the files a variable?  Then we can move them around.

File: modules/WindowsScriptExtension_v1/main.tf [hunk 2]
```diff
@@ -10,25 +10,28 @@ NOTES
...
+                         "https://ansibleteststrgaccnt.blob.core.windows.net/winrmext/ConfigureRemotingForAnsible.ps1"],
+            "storageAccountName": "${var.sa_name}",
```
Comment (CHolmesAtLVS) [inline modules/WindowsScriptExtension_v1/main.tf:31]: Actually, it might be ${var.sa_name}

File: modules/WindowsScriptExtension_v1/main.tf (hunk 1 above)
Comment (pradeeponnada) [reply modules/WindowsScriptExtension_v1/main.tf]: Updated file path as a variable.

File: modules/WindowsScriptExtension_v1/main.tf (hunk 2 above)
Comment (pradeeponnada) [reply modules/WindowsScriptExtension_v1/main.tf:31]: It should be as it is because if we change, we might not be able to authenticate to storage account.

**Review by CHolmesAtLVS — APPROVED** [review]
Looks good.  Thanks Pradeep!

**Pull request merged**